cmd/juju/application/refresh_test.go

```diff
@@ -250,15 +250,15 @@ func (s *UpgradeCharmSuite) TestStorageConstraintsMinFacadeVersion(c *gc.C) {
 	s.apiConnection.bestFacadeVersion = 1
 	_, err := s.runUpgradeCharm(c, "foo", "--storage", "bar=baz")
 	c.Assert(err, gc.ErrorMatches,
-		"updating storage constraints at upgrade-charm time is not supported by server version 1.2.3")
+		"updating storage constraints at refresh time is not supported by server version 1.2.3")
 }
 
 func (s *UpgradeCharmSuite) TestStorageConstraintsMinFacadeVersionNoServerVersion(c *gc.C) {
```

Comment:
s/UpgradeCharmSuite/RefreshSuite/
For UpgradeCharmSuccessStateSuite, UpgradeCharmErrorsStateSuite, BaseUpgradeCharmSuite too.